### PR TITLE
avatar, button, and card examples.  changed package name.  Updated sc…

### PR DESCRIPTION
**Pull request by Magneticmagnum**
…ript to run styleguide from root package

### CODE REVIEW

**Comment by Magneticmagnum** [comment]
REVIEW PLS,

This push includes the new style guide with examples for Avatar, Button, Card.

Test to make sure you can pull down the branch, and start the styleguide server with the following commands:

npm install
npm run bootstrap
npm run styleguide

File: packages/react-atlas-docs/src/components/dialog/Dialog.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { PropTypes } from "react";
+import * as Atlas from 'react-atlas';
+
+export default class Drawer extends React.Component {
+  constructor(props){
+    super(props)
+  }
+  render() {
+    return (
+      <Atlas.Drawer {...this.props}></Atlas.Drawer>
```
Comment (ericsewalson) [inline packages/react-atlas-docs/src/components/dialog/Dialog.js]: this is interesting syntax.  Is this a way we are planning to handle some components going forward using the <Atlas.{component} notation?

File: packages/react-atlas-docs/src/components/dropdownList/DropdownList.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { PropTypes } from "react";
+import * as Atlas from 'react-atlas';
+
+export default class DropdownList extends React.Component {
+  constructor(props){
+    super(props)
+  }
+  render() {
+    return (
+      <Atlas.Button {...this.props}></Atlas.Button>
```
Comment (ericsewalson) [inline packages/react-atlas-docs/src/components/dropdownList/DropdownList.js]: Is it intended to return Button for dropdown?

File: packages/react-atlas-docs/styleguide.config.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  // Use this to select all components in the src/components folder
+
+  // Use this to test a single component.  Change it to the component you are testing and restart the styleguide server
+  // Regex should be: 'src/components/NAME_OF_COMPONENT_FOLDER/[A-Z]*.js'
+	// components: 'src/components/**/[A-Z]*.js',
+	components: 'src/components/button/[A-Z]*.js',
+	defaultExample: true,
+	webpackConfig: {
+		module: {
+			loaders: loaders.all,
```
Comment (ericsewalson) [inline packages/react-atlas-docs/styleguide.config.js]: Not an issue to fix but what loaders does loaders include?  Is this configurable or just a general list of loaders.

**Comment by ericsewalson** [comment]
There aren't many issues aside from the dropdown comment mentioned.  The styleguide looks straight forward and aside from updating all of the readme doc it look good.

**Comment by ericsewalson** [comment]
I will also note the branch did work when I started from a fresh clone of react-atlas.  The npm run bootstrap job did not work in an old clone of the repo.

**Comment by Magneticmagnum** [comment]
The dropdown isn't finishedonoy avatar, card, and button are the only ones that have been successfully migrated.

File: packages/react-atlas-docs/src/components/dropdownList/DropdownList.js (hunk 2 above)
Comment (Magneticmagnum) [inline packages/react-atlas-docs/src/components/dropdownList/DropdownList.js]: Dropdown isn't finished. Thus PR is only for avatar, card, and button. The rest of the components have copied components from Button

File: packages/react-atlas-docs/styleguide.config.js (hunk 3 above)
Comment (Magneticmagnum) [inline packages/react-atlas-docs/styleguide.config.js]: A general list of loaders that is defaulted with styleguidist. We can reconfigure it, but if youre interested, take a look inside node modules 'loaders'

**Pull request merged**